sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_properties.asciidoc

```diff
@@ -338,7 +338,7 @@ struct KernelFunctor {
     a[i] = b[i] + c[i];
   }
 
-  auto get(sycl::ext::oneapi::experimental::properties_tag) {
+  auto get(sycl::ext::oneapi::experimental::properties_tag) const {
```

Comment:
@Pennycook , I think there was some other PR where it was decided that kernel properties can't be runtime. Should we change this to `static constexpr` instead?

Reply:
I agree that kernel properties will always be compile-time.  For a case where a runtime property is required, I think we would use a launch property instead.  Therefore, it seems like it's OK to make this `static constexpr`.

Reply:
I found the discussion @aelovikov-intel was referencing over in https://github.com/intel/llvm/pull/17633, where we ended up adding both `static constexpr` and `auto` versions to deal with compile-time and run-time properties.

Committing to only compile-time properties would solve the problem here and simplify that extension, so I'm also happy to make this `static constexpr`.